compaction/leveled_compaction_strategy.cc

```diff
@@ -237,6 +239,9 @@ leveled_compaction_strategy::get_cleanup_compaction_jobs(table_state& table_s, s
 }
 
 unsigned leveled_compaction_strategy::ideal_level_for_input(const std::vector<sstables::shared_sstable>& input, uint64_t max_sstable_size) {
+    if (!max_sstable_size) {
+        return 1;
+    }
```

Comment:
It's very hard to reason about LCS with max size of 0, perhaps we should forbid it :-) Thanks for this patch.